InnerEye/ML/reports/classification_crossval_report.ipynb

```diff
@@ -0,0 +1,264 @@
+    "        fpr, tpr, thr = roc_curve(labels, model_outputs)\n",
+    "        prc, rec, thr_pr = precision_recall_curve(labels, model_outputs)\n",
+    "        opt_idx = np.argmax(tpr - fpr)\n",
+    "        t50_idx = np.argmin(abs(thr - .5))\n",
+    "        opt_idx_pr = np.argmin(abs(thr_pr - thr[opt_idx]))\n",
+    "        t50_idx_pr = np.argmin(abs(thr_pr - .5))\n",
+    "        auc = roc_auc_score(labels, model_outputs)\n",
+    "        plot_pr_and_roc_curves(labels, model_outputs, axs, plot_kwargs=dict(\n",
+    "            label=f\"Fold {fold_index} (AUC={auc:.2f})\", lw=1))\n",
+    "        tprs.append(np.interp(grid, fpr, tpr))\n",
+    "        prcs.append(np.interp(grid, rec[::-1], prc[::-1]))\n",
+    "        \n",
+    "        axs[0].plot(fpr[opt_idx], tpr[opt_idx], 'o', c=f'C{fold_index}', zorder=1000)\n",
+    "        axs[1].plot(rec[opt_idx_pr], prc[opt_idx_pr], 'o', c=f'C{fold_index}', zorder=1000)\n",
+    "#         axs[0].plot(fpr[t50_idx], tpr[t50_idx], 's', c=f'C{fold_index}', zorder=1000)\n",
+    "#         axs[1].plot(rec[t50_idx_pr], prc[t50_idx_pr], 's', c=f'C{fold_index}', zorder=1000)\n",
+    "        print(\"P(y = 1) =\", labels.mean())\n",
+    "        print(\"P(y_hat > .5) =\", (model_outputs>.5).mean())\n",
+    "        print((labels == (model_outputs>.5)).mean())\n",
+    "    \n",
+    "    tpr_lo, tpr_mid, tpr_hi = quantile(tprs, [.1, .5, .9], axis=0)\n",
+    "    prc_lo, prc_mid, prc_hi = quantile(prcs, [.1, .5, .9], axis=0)\n",
+    "    h1 = axs[0].fill_between(grid, tpr_lo, tpr_hi, color='k', alpha=.2, lw=0)\n",
+    "    h2, = axs[0].plot(grid, tpr_mid, 'k', lw=2)\n",
+    "    ax0handles, ax0labels = axs[0].get_legend_handles_labels()\n",
+    "    ax0handles.append((h1, h2))\n",
+    "    ax0labels.append(\"Median, 80% CI\")\n",
+    "    \n",
+    "    axs[1].fill_between(grid, prc_lo, prc_hi, color='k', alpha=.2, lw=0)\n",
+    "    axs[1].plot(grid, prc_mid, 'k', lw=2)\n",
+    "\n",
+    "    prevalence = sub_df[LoggingColumns.Label.value].mean()\n",
+    "    axs[0].plot([0, 1], [0, 1], ':k', zorder=-1)\n",
+    "    axs[1].axhline(prevalence, ls=':', c='k', zorder=-1)\n",
+    "    axs[0].set_xlabel(\"False positive rate\")\n",
+    "    axs[0].set_ylabel(\"True positive rate\")\n",
+    "    axs[1].set_xlabel(\"Recall\")\n",
+    "    axs[1].set_ylabel(\"Precision\")\n",
+    "    axs[0].legend(ax0handles, ax0labels, frameon=False)"
+   ]
+  }
+ ],
+ "metadata": {
+  "celltoolbar": "Tags",
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
```

Comment:
Can we add a test to make sure this notebook runs correctly? We use nbformat version 4.5, which requires that each cell has a cell id, looks like that is not the case here.